webExtensionApis/study/schema.json

```diff
@@ -1,113 +1,353 @@
 [
   {
     "namespace": "study",
-    "description": "study",
+    "description": "Interface for Shield and Pioneer studies.",
```

Comment:
Isn't this also a generated file from `schema.yaml`? Do we want to include generated files?

Reply:
(See the  answer re fakeApi.  happy to do that as the last patch here.)